clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -8636,6 +8636,7 @@ void SPIRVTranslator::ConstructJob(Compilation &C, const JobAction &JA,
         ",+SPV_INTEL_fpga_cluster_attributes,+SPV_INTEL_loop_fuse"
         ",+SPV_INTEL_long_constant_composite"
         ",+SPV_INTEL_fpga_invocation_pipelining_attributes";
+    ",+SPV_INTEL_fpga_dsp_control";
```

Comment:
```suggestion
        ",+SPV_INTEL_fpga_invocation_pipelining_attributes"
        ",+SPV_INTEL_fpga_dsp_control";
```

Reply:
@AGindinson is causes fail in clang-format, do it anyway?

Reply:
It's just that it seems that without the change, the string is _not_ appended to `INTELExtArg`. Note the semicolon presence before the DSP control substring in the current version of the patch.